test/dummy/config/initializers/mission_control_jobs.rb

```diff
@@ -1,9 +1,11 @@
 require "resque"
 require "resque_pause_helper"
-
 require "solid_queue"
 
-Resque.redis = Redis::Namespace.new "#{Rails.env}", redis: Redis.new(host: "localhost", port: 6379, thread_safe: true)
```

Comment:
thread_safe is noop anyway it seems like

Reply:
Ditto. Reflecting that we're now working with version 5 of the Redis Ruby client. And perhaps causing a regression with version 4. Back out these changes then consider a separate PR to bump the Redis version to `> 5`.